components/tikv_util/src/config.rs

```diff
@@ -872,6 +871,17 @@ mod tests {
             ("23", 23),
             ("1", 1),
             ("1024B", KB),
+            // units with binary prefixes
+            (" 0.5 PiB", PB / 2),
+            ("1PiB", PB),
+            ("0.5 TiB", TB / 2),
+            ("2 TiB", TB * 2),
+            ("0.5GiB ", GB / 2),
+            ("787GiB ", GB * 787),
+            ("0.5MiB", MB / 2),
+            ("3MiB", MB * 3),
+            ("0.5KiB", KB / 2),
+            ("1 KiB", KB),
```

Comment:
So “1K24B" will return "124KB" instead?

Reply:
Yes.

Reply:
I don't think it is a good idea. Report an error would be better in this case. I think you can split str into numeric part and alphabetic part then parse them respectively.

Reply:
Ok, I will update the PR.